homeassistant/components/rflink.py

```diff
@@ -370,6 +373,17 @@ def _async_handle_command(self, command, *args):
             # if the state is true, it gets set as false
             self._state = self._state in [STATE_UNKNOWN, False]
 
+        # Cover options for RFlink
+        elif command == 'close_cover':
+            cmd = 'DOWN'
+
+        elif command == 'open_cover':
+            cmd = 'UP'
+
+        elif command == 'stop_cover':
+            cmd = 'STOP'
+            self._state = True
```

Comment:
When is the state set false, does that never happen?

Reply:
not sure, good question 😊
Will test this without this setting locally